ibis/backends/tests/test_array.py

```diff
@@ -4,6 +4,19 @@
 import ibis
 
 
+@pytest.mark.xfail_unsupported
+@pytest.mark.skip_missing_feature(
```

Comment:
How does this decorator work?

Reply:
The default feature test flags are defined here:
https://github.com/ibis-project/ibis/blob/b78ea0cd4ab7e955000e3252bd21e66981420d25/ibis/backends/tests/base.py#L63-L73

Each backend's `tests/conftest.py` can override which feature test flags should be enabled for that backend:
https://github.com/ibis-project/ibis/blob/b78ea0cd4ab7e955000e3252bd21e66981420d25/ibis/backends/sqlite/tests/conftest.py#L11-L16

([And here is where the behavior of the decorator is actually implemented)](https://github.com/ibis-project/ibis/blob/b78ea0cd4ab7e955000e3252bd21e66981420d25/ibis/backends/tests/conftest.py#L87-L96)